### PR TITLE
[scala] Make ensime use stable version as unstable is for ensime devs

### PR DESCRIPTION
**Pull request by d1egoaz**
Make ensime use stable version as unstable is for ensime devs.

Ensime unstable is not intended for general use and will cause problems.
see https://github.com/ensime/ensime-emacs/issues/553

### CODE REVIEW

File: layers/+lang/scala/README.org [hunk 1]
```diff
@@ -61,8 +69,8 @@ project's =project/build.properties=. For example,
    ensimeConfig= and =sbt ensimeConfigProject= at the shell.
 2. Run =M-x ensime= within Emacs to start an ENSIME session.
 
-Each Scala project uses a dedicated ENSIME session, so you only need to run =M-x
-ensime= once per project. Any Scala files you create or visit within the project
+Each Scala project uses a dedicated ENSIME session, so you only need to
+run =SPC SPC ensime= once per project. Any Scala files you create or visit within the project
```
Comment (LemmingAvalanche) [inline layers/+lang/scala/README.org]: Should be:

```
... run ~SPC SPC ensime~ ...
```

Instead, since it is a keybinding.

**Comment by diego-alvarez-hs** [comment]
up

**Comment by diego-alvarez-hs** [comment]
bump, @TheBB

**Comment by TheBB** [comment]
Thanks! Cherry-picked in develop. You can safely delete your branch.

I put it in the java layer readme too, since ensime can be found there as well.

**Pull request closed without merging**